src/main/java/org/ow2/proactive/catalog/graphql/bean/argument/WhereArgs.java

```diff
@@ -23,18 +23,39 @@
  * If needed, contact us to obtain a release under GPL Version 2 or 3
  * or a different license than the AGPL.
  */
-package org.ow2.proactive.catalog.graphql.schema.type;
+package org.ow2.proactive.catalog.graphql.bean.argument;
 
 import lombok.Data;
-import lombok.ToString;
+import lombok.NoArgsConstructor;
 
 
 /**
  * @author ActiveEon Team
- * @since 08/06/2017
+ * @since 09/06/2017
  */
+@NoArgsConstructor
 @Data
-@ToString
-public class Bucket {
+public class WhereArgs<T> {
+
+    protected T eq;
+
+    protected T ne;
+
+    protected T gt;
+
+    protected T gte;
+
+    protected T lt;
+
+    protected T lte;
+
```

Comment:
Could you please add a comment before the constructor of WhereArgs.java to explain what does this eq, ne, gt ... etc.. mean